.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -2,6 +2,22 @@ name: Bug report
 description: Reporting an issue
 labels: ["bug"]
 body:
+  - type: dropdown
+    attributes:
+      label: Operating System
+      description: On which operating system do you have a problem?
+      options:
+        - Windows
+        - MacOS
+        - Linux
```

Comment:
I would also include mobile os (android and ios) and an option for other that they can specify.

Reply:
I got this with your request. I don't know if ot's possible to hide/show other field dynamically :/

![image](https://user-images.githubusercontent.com/40738104/177050478-bd097ecd-ba7a-4025-a3a6-9444c5883aa9.png)